framework/src/media/audio/audio_manager.c

```diff
@@ -453,22 +454,23 @@ static audio_manager_result_t get_audio_volume(int fd, audio_config_t *config, a
 		caps_desc.caps.ac_format.hw = AUDIO_FU_INP_GAIN;
 	}
 
-	ret = ioctl(fd, AUDIOIOC_GETCAPS, (unsigned long *)&caps_desc.caps);
-	if (ret < 0) {
-		meddbg("Fail to ioctl AUDIOIOC_GETCAPS\n");
-		return AUDIO_MANAGER_DEVICE_FAIL;
+	if ((ioctl(fd, AUDIOIOC_GETCAPS, (unsigned long *)&caps_desc.caps)) < 0) {
```

Comment:
please check here.
below is avalable for only return is -ENOSYS

Reply:
alc5658.c is modified so that this ioctl returns success though though exclude configs are set.